qa/rpc-tests/wallet_shieldcoinbase.py

```diff
@@ -33,6 +36,9 @@ def setup_network(self, split=False):
         self.sync_all()
 
     def run_test (self):
+        # create one zaddr that is the target of all shielding
+        myzaddr = self.test_init_zaddr(self.nodes[0])
+
```

Comment:
Doing this at the start of the test, before mining any blocks, causes the Sprout variant to fail with:
```
JSONRPC error: Error: Creating a Sprout address during initial block download is not supported.
```

Move this to just after the post-mining balance checks, before `do_not_shield_taddr` is created.